src/Codeception/Module/Yii2.php

```diff
@@ -322,7 +326,7 @@ private function loadFixtures($test)
             $this->debugSection('Fixtures', 'Opened database connection: ' . $event->sender->dsn);
             $connections[] = $event->sender;
         });
-        if (empty($this->loadedFixtures)
+        if (($force || empty($this->loadedFixtures))
```

Comment:
Why not just check config directly? 
`if ($this->_getConfig('forceLoadFixtures') && ...`

That way you don't need to change the function and its calls.